modules/manta/germline/main.nf

```diff
@@ -8,11 +8,12 @@ process MANTA_GERMLINE {
         'quay.io/biocontainers/manta:1.6.0--h9ee0642_1' }"
 
     input:
-    tuple val(meta), path(input), path(input_index)
+    tuple val(meta), path(input)
+    path index
     path fasta
-    path fai
-    path target_bed
-    path target_bed_tbi
+    path fasta_fai
+    tuple val(bed_meta), path(target_bed), path(target_bed_tbi)
```

Comment:
Is it possible to run multiple regions per patient? In that case target_bed should go with the input file. What is the content of the bed_meta, I haven't seen this yet in the other modules?

Reply:
target_bed is not defined for each patient rather it is defined for all the patients in a case. 

My rationale behind having a bed_meta is that the target_bed file need not have the same information in its meta as the alignment files. For instance, the id here could be something other than the case/sample id. So defining it as bed_meta avoids those conflicts.

Reply:
On second thought, bed_meta isn't really used in the module so I could just get rid of it, and the user can pass a tuple containing the path to target_bed and target_bed_tbi. What do you think?

Reply:
Yes I think this is sensible.

Reply:
Do you know if it is possible to parallelize manta about regions. I.e. for each patient run manta in aprallel per lets say chromosome?

Reply:
As far as I know, the option to do that is not baked into manta so I think that should be handled upstream.

Reply:
hm yes, ok. Just to be sure, my thought was something similar to what Haplotypecaller does, were you can ran all regions parallely and join them afterwards. (The module itself wouldn't do any parallelization):

chr1 -> run_manta(patient1,chr1) -> results_pat1_chr1
chr1 -> run_manta(patient2,chr1)-> results_pat2_chr1
chr2 -> run_manat(patient1,chr2)-> results_pat1_chr2
chr2 -> run_manat(patient2,chr2)-> results_pat2_chr2

join(results_pat1_chr1, results_pat1_chr2)
join(results_pat2_chr1, results_pat2_chr2)

Reply:
ah, I haven't tried it, but I don't see a reason for why it would not work because the information doesn't traverse chromosomes.